app/services/IdentityService.scala

```diff
@@ -178,59 +180,12 @@ trait IdentityApiClient {
   def consentEmail: Email => Future[WSResponse]
 }
 
-object IdentityApiClient extends IdentityApiClient with LazyLogging {
-
-  import PersonalDataJsonSerialiser._
-
-  lazy val identityEndpoint = Config.Identity.baseUri
-  lazy val metrics = new IdApiMetrics(Config.stage)
-
-  implicit class FutureWSLike(eventualResponse: Future[WSResponse]) {
-    /**
-     * Example of ID API Response: `{"status":"error","errors":[{"message":"Forbidden","description":"Field access denied","context":"privateFields.billingAddress1"}]}`
-     *
-     * @return
-     */
-    private def applyOnSpecificErrors(reasons: List[String])(block: WSResponse => Unit): Unit = {
-      def errorMessageFilter(error: JsValue): Boolean =
-        (error \ "message").asOpt[JsString].exists(e => reasons.contains(e.value))
+class IdentityApiClientImpl(wsClient: WSClient) extends IdentityApiClient with LazyLogging {
```

Comment:
here I created a class so I can inject the ws client, but I only put the stuff that needs a ws client into it, and left the other stuff in the object (which is now below)